Sources/Datadog/Datadog.swift

```diff
@@ -152,8 +140,24 @@ public class Datadog {
             )
         }
 
+        if configuration.rumEnabled {
+            rum = RUMFeature(
+                directory: try obtainRUMFeatureDirectory(),
+                configuration: validConfiguration,
+                performance: performance,
+                mobileDevice: mobileDevice,
+                httpClient: httpClient,
+                dateProvider: dateProvider,
+                userInfoProvider: userInfoProvider,
+                networkConnectionInfoProvider: networkConnectionInfoProvider,
+                carrierInfoProvider: carrierInfoProvider
+            )
+        }
+
         LoggingFeature.instance = logging
         TracingFeature.instance = tracing
+        RUMFeature.instance = rum
+
         TracingAutoInstrumentation.instance = TracingAutoInstrumentation(with: configuration)
```

Comment:
can you please add `configuration.rumEndpoint.url` to `excludedURLs` within this method?
you will then need to fix a test case in `TracingAutoInstrumentationTests` i guess

Reply:
Ah, sure - done 👍. Good catch 👌.